docs/conf.py

```diff
@@ -125,6 +125,14 @@
     # Setup.py files of the providers
     "_api/airflow/providers/*/*setup/index.rst",
     "_api/airflow/providers/*/*/*setup/index.rst",
```

Comment:
```suggestion
```
I don't think we need these anymore, right?

Reply:
I am afraid to try (as I will find it after full build. But it is worth trying.

Reply:
Yep. That worked.